Dockerfile-proxy

```diff
@@ -37,6 +37,7 @@ COPY --from=fetch /build/proxy-version /usr/lib/linkerd/linkerd2-proxy-version.t
 COPY --from=fetch /build/linkerd2-proxy /usr/lib/linkerd/linkerd2-proxy
 COPY --from=fetch /build/linkerd-await /usr/lib/linkerd/linkerd-await
 COPY --from=golang /out/proxy-identity /usr/lib/linkerd/linkerd2-proxy-identity
+COPY --from=debian:buster-20210208-slim /bin/sleep /bin/sleep
```

Comment:
we can go as recent as `bullseye-20210816-slim` here

Reply:
buster-20210208-slim is used by the linkerd-cni container and the debug container.  I'm inclined to keep these on the same version for consistency and upgrade them all at once unless there's a reason not to.